Library/Formula/srmio.rb

```diff
@@ -15,7 +15,7 @@ class Srmio < Formula
 
   def install
     if build.head?
-      system "chmod u+x genautomake.sh"
+      chmod 1, "genautomake.sh"
```

Comment:
That makes sense. I had thought that absolute mode was preferred over symbolic mode (partly because of the prevalence of absolute mode in some of the `chmod` discussions I'd read), but I really don't know. Thanks for the feedback - I'm a very new Homebrew contributor. I'll go ahead and make this change.